src/state/nftMarket/reducer.ts

```diff
@@ -97,6 +98,16 @@ export const fetchUserNfts = createAsyncThunk<
   NftTokenSg[],
   { account: string; profileNftWithCollectionAddress: TokenIdWithCollectionAddress; collections: ApiCollections }
 >('nft/fetchUserNfts', async ({ account, profileNftWithCollectionAddress, collections }) => {
+  const getCategoryForNftWithMarketData = (marketNft: NftTokenSg): NftLocation => {
```

Comment:
Changes in this action are all attaching the different nftLocations (in wallet, on sale, profile pic) to the `NftTokenSg` user nft objects